docs/rules/prefer-arrow-callback.md

```diff
@@ -1,77 +1,100 @@
-# Suggest using arrow functions as callbacks. (prefer-arrow-callback)
+# Suggest using ES6 arrow functions to describe callbacks (prefer-arrow-callback)
```

Comment:
The word `describe` doesn't quite feel right to me. What do you think of making it something like `Suggest using ES6 arrow functions for callbacks (prefer-arrow-callback)`?

Reply:
Can I change the metadata property as part of this same PR? or do I need to create a new one to make them match?

Reply:
Please do change the metadata on this same PR if you can. Thanks!